src/main/scala/org/hammerlab/guacamole/commands/jointcaller/Input.scala

```diff
@@ -16,10 +17,11 @@ import org.hammerlab.guacamole.readsets.{SampleId, SampleName}
  * @param analyte rna or dna
  */
 case class Input(index: SampleId,
-                 sampleName: SampleName,
```

Comment:
We already do! Its name is `…jointcaller.Input`.

But seriously; defensively naming things to try to avoid conflicts at possible `import` sites is not good practice.